### ISSUE TITLE
PikerField Selection Change Handler

### ISSUE THREAD

**Issue by aliEsmaelFard** [enisn/UraniumUI#594]
With the help of which handler we can understand the Selections Change?  
  <material:PickerField Title="Select View Type"
                        ItemsSource="{Binding ControlTypeList}" 
                        Margin="0,8"  />

**Comment by enisn** [comment]
You can use binding for `SelectedItem`. It'll trigger the setter of your property when it's changed.

```xml
<material:PickerField Title="Select View Type"
ItemsSource="{Binding ControlTypeList}"
Margin="0,8"
SelectedItem="{Binding SelectedItem}" />
```

```csharp
public class YourViewModel : ViewModelBase
{
    object selectedItem;
    public object SelectedItem 
    { 
        get => selectedItem;
        set
        {
            selectedItem = value;
            // Here you can handle the changes...
        }
    }
}
```


I'll consider adding events & commands for selection changes, but in the current version it's possible in this way

**Comment by aliEsmaelFard** [comment]
thank you. i have another questions why MultiplePickerField  not work in windows?

**Comment by enisn** [comment]
> thank you. i have another questions why MultiplePickerField not work in windows?

It uses a dialog implementation, did you configure `UraniumUI.Dialogs.CommunityToolkit` or `UraniumUI.Dialogs.Mopups`?

They come with a simple configuration: 
https://enisn-projects.io/docs/en/uranium/latest/dialogs/Index

**Comment by aliEsmaelFard** [comment]
Still Not Working Even In android Device

**Comment by aliEsmaelFard** [comment]
And I Check Your Demo Not Working In There too

**Comment by enisn** [comment]
I found the problem and fixed with https://github.com/enisn/UraniumUI/pull/598

It'll be included in the next release

**Comment by aliEsmaelFard** [comment]
when its released?

**Comment by enisn** [comment]
It was shipped in [v2.8.0](https://github.com/enisn/UraniumUI/releases/tag/v2.8.0) at the last minute 👍

**Comment by aliEsmaelFard** [comment]
thanks
